### PR TITLE
Misc fluent fixes

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs [hunk 1]
```diff
@@ -26,9 +26,10 @@ public IngamePowerBarLogic(Widget widget, ModData modData, World world)
 			var powerManager = world.LocalPlayer.PlayerActor.Trait<PowerManager>();
 			var powerBar = widget.Get<ResourceBarWidget>("POWERBAR");
 
+			var powerUsageText = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]:
```suggestion
			var powerUsage = modData.Translation.GetString(PowerUsage);
```

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs [hunk 2]
```diff
@@ -26,9 +26,10 @@ public IngamePowerBarLogic(Widget widget, ModData modData, World world)
...
 			powerBar.GetProvided = () => powerManager.PowerProvided;
 			powerBar.GetUsed = () => powerManager.PowerDrained;
-			powerBar.TooltipFormat = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
+			powerBar.TooltipFormat = powerUsageText;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]:
```suggestion
			powerBar.TooltipFormat = powerUsage + ": {0}/{1}";
```

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs (hunk 1 above)
Comment (PunkPun) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: sure

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs (hunk 2 above)
Comment (PunkPun) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: we do we need to concatenate every frame?

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs (hunk 1 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: See the second change.

File: OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs [hunk 3]
```diff
@@ -92,7 +92,7 @@ public SpawnSelectorTooltipLogic(Widget widget, ModData modData, TooltipContaine
 			flag.IsVisible = () => playerFaction != null;
 			flag.GetImageCollection = () => "flags";
 			flag.GetImageName = () => playerFaction;
-			team.GetText = () => teamMessage.Update(playerTeam);
+			team.GetText = () => playerTeam > 0 ? teamMessage.Update(playerTeam) : "";
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs:95]: This should not be visible when `playerTeam == 0`?
Edit: Just saw the commit message, so nvm. Could use `string.Empty` though.

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs [hunk 4]
```diff
@@ -26,9 +26,10 @@ public IngamePowerBarLogic(Widget widget, ModData modData, World world)
 			var powerManager = world.LocalPlayer.PlayerActor.Trait<PowerManager>();
 			var powerBar = widget.Get<ResourceBarWidget>("POWERBAR");
 
+			var powerUsage = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
 			powerBar.GetProvided = () => powerManager.PowerProvided;
 			powerBar.GetUsed = () => powerManager.PowerDrained;
-			powerBar.TooltipFormat = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
+			powerBar.TooltipFormat = powerUsage;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: `TooltipFormat` is just a string, not a function, so setting this here once is ok.

Comment (PunkPun) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: you're right! I tackled the wrong translation, the issue was in IngamePowerCounterLogic

File: OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs (hunk 3 above)
Comment (PunkPun) [reply OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs:95]: Do we use `string.Empty`? I have nothing against it but I just have don't remember ever seeing it

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs (hunk 2 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: `{0}/{1}` could change?

**Review by Mailaender — APPROVED** [review]

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs (hunk 2 above)
Comment (PunkPun) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs]: https://github.com/OpenRA/OpenRA/pull/20269#discussion_r965586810

File: OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs (hunk 3 above)
Comment (abcdefg30) [reply OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs:95]: It doesn't make much of a difference in practice, so we can stay with `""`.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/31aeba52f1fc3fc8332217addc999fa47dcda5fe...31eec5b33bfdec67bdb5ff4d02c3bfab65358080)

**Pull request merged**